k8_cortx_cloud/solution.yaml

```diff
@@ -3,12 +3,12 @@ solution:
   secrets:
     name: cortx-secret
     content:
-      kafka_admin_secret: Seagate@123
-      consul_admin_secret: Seagate@123
-      common_admin_secret: Seagate@123
-      s3_auth_admin_secret: cortxadmin
-      csm_auth_admin_secret: seagate2
-      csm_mgmt_admin_secret: Cortxadmin@123
+      kafka_admin_secret:
```

Comment:
we'll want to make the same updates to solution_stub.yaml as well.

Reply:
Good catch!  Will do.

Reply:
Done.